conan/tools/cmake/toolchain/blocks.py

```diff
@@ -824,14 +828,21 @@ def context(self):
 
         system_name, system_version, system_processor = self._get_cross_build()
 
+        # This is handled by the tools.apple:sdk_path and CMAKE_OSX_SYSROOT in Apple
+        cmake_sysroot = None
+        if not is_apple_os(self._conanfile.settings.get_safe("os")):
```

Comment:
why? As this is a defined conf, is it really necessary to not apply it the OS is an apple one?
Is it possible that there there are some scenario (like cross-building or the like) that would want to use the ``sysroot``? Wouldn't it be enough that the user don't define ``tools.build:sysroot`` in their profile (or inside the package, the package recipe would know)

Reply:
Yes, maybe it's better not to do this, I'll remove the check 👍